bundle/config/mutator/translate_paths_test.go

```diff
@@ -455,3 +456,143 @@ func TestPipelineFileDoesNotExistError(t *testing.T) {
 	err := mutator.TranslatePaths().Apply(context.Background(), bundle)
 	assert.EqualError(t, err, "file ./doesnt_exist.py not found")
 }
+
+func TestSparkPythonTaskJobWithNotebookSourceError(t *testing.T) {
```

Comment:
Please use prefix `TestJob` like the other tests, then one can run all job related tests with the same prefix.

Same for the other tests below.